shared/macros-oval.jinja

```diff
@@ -150,7 +150,7 @@
 #}}
 {{%- set regex = prefix_regex+parameter+separator_regex -%}}
 {{%- else %}}
-{{%- set regex = prefix_regex+parameter+separator_regex+"(.*)[\s]*(?:|(?:#.*))?$" -%}}
+{{%- set regex = ".*(?:\\n\s*[^[\s].*)*\s*"+prefix_regex+parameter+separator_regex+"(.*)[\s]*(?:|(?:#.*))?$" -%}}
```

Comment:
This seems to complex, it deserves a comment.

Reply:
I have proposed an alternate fix, https://github.com/ComplianceAsCode/content/pull/4647 :)